src/Actions/Role/UpdateUserRoles.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace FluxErp\Actions\Role;
+
+use FluxErp\Actions\FluxAction;
+use FluxErp\Http\Requests\EditUserRoleRequest;
+use FluxErp\Models\Role;
+use FluxErp\Models\User;
+
+class UpdateUserRoles extends FluxAction
+{
+    protected function boot(array $data): void
+    {
+        parent::boot($data);
+
```

Comment:
revert $this->setData() because rules do not guarantee that data['assign'] ist set